packages/terra-base/tests/wdio/base-spec.js

```diff
@@ -26,4 +32,20 @@ describe('Base', () => {
       expect($('div').getText()).to.contain('No Translations String');
     });
   });
+
+  describe('Anchors', () => {
+    beforeEach(() => browser.url('/#/tests/base-tests/anchors'));
+
+    Terra.should.beAccessible({ viewports, rules: ignoredA11y });
+    Terra.should.matchScreenshot({ viewports });
+    Terra.should.themeEachCustomProperty({
+      '--terra-link-color': '#0065a3',
```

Comment:
All of your screen shots look the same. You'll need to actually move the mouse to test hover, send the tab key to test focus, etc. Also, these theme tests should be using values that are different from the default. You should be able to see in the screenshot that the values were modified.

Reply:
Talked with @msaikaushik1992 on this, we are going to add classes to the different selectors to make it easier to test the link pseudo class.

e.g.
```
a:focused,
a.is-focused {
/* ... focus styles ... */
}
```

This should allows to test the varying link states without needing to move the mouse or click on links to view visited and active state. The theme tests will be updated with different values so we can see the difference in the screenshots.

Reply:
Updated https://github.com/cerner/terra-core/pull/1171/commits/b0b8cb91ec5f7dcb08e63cec24763df41004c2df